scripts/rust_g.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+test_rust=$(ls $HOME/rust_g-$RUST_G_VERSION | grep "rust_g")
```

Comment:
Technically `if [ ! -f $HOME/rust_g-$RUST_G_VERSION/rust_g ]; then` is a more appropriate check. But ye.